extension/otto/otto_test.go

```diff
@@ -1779,6 +1781,100 @@ var _ = Describe("Otto extension manager", func() {
 		})
 	})
 
+	Describe("Using gohan_get_env builtin", func() {
+
+		It("Should return proper environmental variable", func() {
+			os.Setenv("GOHAN_TEST_GET_ENV", "123")
+			extension, err := schema.NewExtension(map[string]interface{}{
+				"id": "test_extension",
+				"code": `
+					gohan_register_handler("test_event",
+					 	function(context) {
+					 		context.resp = gohan_get_env("GOHAN_TEST_GET_ENV", "");
+						}
+					);
+					`,
+				"path": ".*",
+			})
+			Expect(err).ToNot(HaveOccurred())
+			extensions := []*schema.Extension{extension}
+			env := newEnvironment()
+			context := map[string]interface{}{}
+			Expect(env.LoadExtensionsForPath(extensions, timeLimit, timeLimits, "test_path")).To(Succeed())
+			Expect(env.HandleEvent("test_event", context)).To(Succeed())
+			Expect(context["resp"]).To(Equal("123"))
+
+			os.Unsetenv("GOHAN_TEST_GET_ENV")
```

Comment:
it won't be called if the test fails, potentially harming other tests. move to Before/AfterEach